core/domain/state_domain.py

```diff
@@ -229,6 +237,10 @@ def validate(
                     'to be <skill_id>-<misconception_id>, received %s'
                     % self.tagged_skill_misconception_id)
 
+        if len(self.rule_specs) == 0 and validation_from_exploration:
+            raise utils.ValidationError(
+                'There must be at least one rule for each answer group.')
```

Comment:
Please add this error to the "Raises" section in the docstring.

Reply:
Done